runbot_send_email/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Use the png of the new template:
https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

Check if there are something new to add here